backend/Tests/Api/GraphQL/__snapshots__/committed-schema.verified.graphql

```diff
@@ -2172,8 +2194,11 @@ enum ContractVersion {
   V1
 }
 
+"Enumeration of the types of credentials."
 enum CredentialDeploymentTransactionType {
+  "Initial credential is a credential that is submitted by the identity\nprovider on behalf of the user. There is only one initial credential\nper identity."
   INITIAL
+  "A normal credential is one where the identity behind it is only known to\nthe owner of the account, unless the anonymity revocation process was\nfollowed."
```

Comment:
```suggestion
  "A normal credential is one where the identity behind it is only known to\nthe owner of the account, unless the identity disclosure process has been initiated."
```

Reply:
Same as above, these are generated from types in .NET SDK so will be fixed in a different PR.